catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfPreStoragePlugin.java

```diff
@@ -207,9 +210,14 @@ private ContentItem createDerivedImage(String id, String qualifier, BufferedImag
         return null;
     }
 
-    private String buildDerivedImageTitle(String title, String qualifier) {
+    protected String buildDerivedImageTitle(String title, String qualifier) {
         String rootFileName = FilenameUtils.getBaseName(title);
-        return String.format(DERIVED_IMAGE_FILENAME_PATTERN, qualifier, rootFileName, JPG);
+
+        return (StringUtils.isNotBlank(rootFileName) &&
+                StringUtils.isNotBlank(rootFileName.replaceAll("[^A-Za-z0-9]", ""))) // any human readable content
+                ? String.format(DERIVED_IMAGE_FILENAME_PATTERN, qualifier,
+                        rootFileName.replaceAll(INVALID_FILENAME_CHARACTER_REGEX, ""), JPG)
+                : String.format("%s.%s", qualifier, JPG);
```

Comment:
I was not sure about this and sent this question to @dcruver yesterday. He is also a reviewer on this PR.